README.md

```diff
@@ -82,7 +85,7 @@ the snapshot manifest will expire.
 Stages an appropriate `timestamp` manifest. If a `snapshot` manifest is staged,
 it must be fully signed.
 
-#### `tuf sign ROLE`
+#### `tuf sign <manifest>`
```

Comment:
Can we call this `metadata` for consistency?

Reply:
Thanks, @mnm678 👍 Actually, while doing that I've noticed that this inconsistency is also present throughout the code/documentation (raised it in the #tuf slack channel today). I'll open a separate PR where I fix this inconsistency and update every use of `manifest` against `metadata` throughout the repo